docroot/modules/custom/foia_webform/src/FoiaSubmissionApiService.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
+
+use Drupal\foia_webform\AgencyLookupServiceInterface;
+use Drupal\webform\WebformSubmissionInterface;
+use GuzzleHttp\ClientInterface;
+use Drupal\node\Entity\Node;
+
+/**
+ * Class FoiaSubmissionApiService.
+ *
+ * @package Drupal\foia_webform\Plugin\QueueWorker
+ */
+class FoiaSubmissionApiService implements FoiaSubmissionServiceInterface {
+
+  private $apiURL;
```

Comment:
Let's omit this as a property on the class. Since we're only using it within the same method that we set it in, we don't buy much by having it as a property.